src/components/MeetOurTeam/MeetOurTeam.jsx

```diff
@@ -0,0 +1,31 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faLinkedin } from '@fortawesome/free-brands-svg-icons';
+
+function MeetOurTeam({teamMembers}){
```

Comment:
Beautiful work on passing the array as props!🙌
This allows us to be able to reuse the component for other data!🎊
Awesome work on implementing the best practice!!💪

Reply:
Thank you @AllanSaleh ! I wouldn't have done it without you